apps/src/templates/teacherDashboard/AddSectionDialog.jsx

```diff
@@ -17,6 +17,23 @@ import {
 } from './teacherSectionsRedux';
 import ParticipantTypePicker from './ParticipantTypePicker';
 import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import {navigateToHref} from '@cdo/apps/utils';
+import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
+import experiments from '@cdo/apps/util/experiments';
+
+// Checks if experiment is enabled and navigates to the new section setup page
+// if both params are non-null.
+const redirectToNewSectionPage = (participantType, loginType) => {
+  if (
+    experiments.isEnabled('sectionSetupRefresh') &&
```

Comment:
Could you by any chance show what it looks like if you try to access the url without those params? Does it try to load (and look like it's spinning) and then just bounce you back to the page you were on before?

Reply:
no, it will simply redirect back to the homepage (I'm not sure why the application dashboard is flashing there -- I think it might be something about how we load the homepage for admins)

https://user-images.githubusercontent.com/46464143/223226199-e26991ef-9e8e-40cc-b242-b2e69f432e72.mov